app/presenters/formats/edition_format_presenter.rb

```diff
@@ -97,5 +98,9 @@ def major_change?
     def public_updated_at
       edition.public_updated_at || edition.updated_at
     end
+
+    def last_edited_by_editor_id
+      edition.created_by&.uid
+    end
```

Comment:
Maybe worth adding a comment here to explain that we do not really have access to information who edited the edition last, so we rely on created_by.